storybook/config.js

```diff
@@ -6,11 +6,132 @@ import i18n from '@/i18n'
 
 import vocabularyTheme from './theme'
 
+// Configuration of addon-viewport
+
+const newViewports = {
+
+  // Mobile devices
+
+  iPhoneX: {
+    name: 'iPhoneX',
+    styles: {
+      width: '375px',
+      height: '812px'
+    }
```

Comment:
All viewports should also define a key `type` which can be `'desktop'`, `'mobile'` or `'tablet'`.